python/Pipfile

```diff
@@ -0,0 +1,15 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+maturin = "==0.12.11"
+datafusion = "==0.5.1"
```

Comment:
upgrade to new version after PR https://github.com/datafusion-contrib/datafusion-python/pull/46